doc/2/api/essentials/codes/index.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Why this file name is not `error-codes` instead? IMHO, it will be more explicit in the URL